partcad/src/partcad/context.py

```
@@ -69,8 +73,9 @@ def __enter__(self, *_args):
         def __exit__(self, *_args):
             self.lock.release()
 
-    def __init__(self, root_path=None, search_root=True):
+    def __init__(self, root_path=None, search_root=True, transaction=None):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)